kahuna/public/js/upload/file-uploader.html

```diff
@@ -1,17 +1,14 @@
 <form class="file-uploader">
-    <div ui:event-share="click"
-         ui:event-share-thief=".file-uploader__select-files"
-         ui:event-share-victim=".file-uploader__file">
-        <!-- you cannot bind ng:model="files" nor use ng:change:
-        https://github.com/angular/angular.js/issues/1375 -->
-        <input class="file-uploader__file"
-               name="files" type="file"
-               multiple
-               ui:file ui:file-change="fileUploader.uploadFiles" />
+    <!-- you cannot bind ng:model="files" nor use ng:change:
+    https://github.com/angular/angular.js/issues/1375 -->
+    <input class="file-uploader__file"
+           name="files" type="file"
+           multiple
+           gr:file-change="ctrl.uploadFiles" />
 
-        <button class="file-uploader__select-files button">
-            <i class="material-icons">file_upload</i>
-            <span class="file-uploader__select-files--label">upload</span>
-        </button>
-    </div>
+    <button class="file-uploader__select-files button"
+            gr:track-click="Upload button">
```

Comment:
What value would that add? Can't we just search for Witness images in the Grid and we know how many were done?